packages/artifact/docs/artifact-contributions.md

```diff
@@ -0,0 +1,30 @@
+# Contributions
+
+This package is used internally by the v2+ versions of [upload-artifact](https://github.com/actions/upload-artifact) and [download-artifact](https://github.com/actions/download-artifact). Any changes or updates to this package will propagate updates to these actions so it is important that major changes or updates get properly tested.
```

Comment:
It might be worth specifying that this package could/will be used by other actions as well.